master/reference/advanced/etcd-rbac/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Setting up etcd certificates for RBAC
+---
+
+## Overview
+
+When using etcd it is a good idea to protect the data stored there.  This is
```

Comment:
I don't think this really summarizes the purpose of the guide.  I think it's to:
-  Restrict the operations that support staff are able to perform to minimize accidental corruption or deletion of Calico data
-  if the cluster is used by multiple orchestrators (Kubernetes, docker) to lock down access to each orchestrators specific set of data
-  Restrict the read/write access of the various Calico components as an additional safety measure